web/test/mock/config.js

```diff
@@ -18,6 +18,11 @@
         },
         record: {
             limit: 50
+        },
+        recordType: {
+            visible: false,
```

Comment:
Oops: I renamed it from display to visible before pushing, but forgot to change the one reference.